src/main/java/org/apache/commons/configuration2/convert/PropertyConverter.java

```diff
@@ -70,6 +70,9 @@ public final class PropertyConverter {
     /** The fully qualified name of {@code javax.mail.internet.InternetAddress} */
     private static final String INTERNET_ADDRESS_CLASSNAME = "javax.mail.internet.InternetAddress";
 
+    /** The fully qualified name of {@code jakarta.mail.internet.InternetAddress} */
```

Comment:
Maybe rename the other constant to have a JAVAX postfix?

Reply:
Good idea. See #188